src/v3/src/transformer/phone/phoneEnrollmentTransformer.test.ts

```diff
@@ -67,7 +60,10 @@ describe('PhoneEnrollmentTransformer Tests', () => {
     expect(updatedFormBag).toMatchSnapshot();
   });
 
-  it('should validate phoneNumber format', () => {
+  // TODO: re-visit when add phone number client side validation
+  // it's not safe to use one client side regext to validate, since phone number standard varies by regions
+  // before a good hint is available from idx response, it would be safe to just depend on backend validation
+  it.skip('should validate phoneNumber format', () => {
```

Comment:
What is the v2 behavior? If we currently attempt to do client side phone validation, we still should attempt to do so in v3 but let the backend do final validation.

Reply:
v2 only do client side validation based on `required` attr

Reply:
If that's the case, we can delete client-side validation code for phone and remove this test